crates/cibse_guide_e/src/chapter_6/equation_6_57/openfire_runner.rs

```diff
@@ -0,0 +1,178 @@
+pub mod integration_tests;
+
+use framework::method::calculation::Calculation;
+use framework::method::calculation::CalculationComponent;
+use framework::method::form::{Form, FormStep};
+use framework::method::parameter::ArcParameter;
+use framework::method::parameter::ParameterTrait;
+use framework::method::parameter::Parameters;
+use framework::method::parameter::builder::ParamBuilder;
+use framework::method::runner::MethodRunner;
+use framework::method::tag::Tag;
+use framework::method::test::IntegrationTests;
+use framework::method::validation::ParameterError;
+use framework::method::{Method, step::Step};
+
+use crate::CIBSEGuideE;
+
+use std::sync::{Arc, RwLock};
+use std::vec;
+
+struct Symbols {
+    z_fo: &'static str,
+    r: &'static str,
+    w: &'static str,
+    h_o: &'static str,
+}
+
+const SYMBOLS: Symbols = Symbols {
+    z_fo: "z_{fo}",
+    r: "R",
+    w: "w",
+    h_o: "h_{o}",
+};
+
+#[derive(Default)]
+pub struct Chapter6Equation57Runner;
+
+impl MethodRunner for Chapter6Equation57Runner {
+    fn name(&self) -> String {
+        "Height of the flame above the top of the opening".to_string()
+    }
+    fn reference(&self) -> &dyn framework::method::runner::Reference {
+        &CIBSEGuideE::Chaptersix(crate::chapter_6::Chapter6Method::Equation6_57)
```

Comment:
```suggestion
        &CIBSEGuideE::ChapterSix(crate::chapter_6::Chapter6Method::Equation6_57)
```
I don't think this is part of this PR, but can we change this just to keep it consistent.

Reply:
Addressed.